recipes/sagemaker-code-editor/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "sagemaker-code-editor" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz  # [linux64]
+  sha256: 9effda827c45e1fd8dfc534ce310791ee46070d818ede89abf19e3ec9c8906d0  # [linux64]
+  folder: sagemaker-code-editor
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - sysroot_linux-64 ==2.17
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+    - nodejs ==18.18.2
+    - openssl ==3.1.4
+    - jq
+    - krb5 ==1.21.2  # [linux]
+    - libsecret  # [linux]
+    # node-gyp is broken with python 3.12 - https://github.com/nodejs/node-gyp/issues/2869, so pulling in distutils
+    - setuptools
+    - yarn <2
+  host:
+    - nodejs ==18.18.2
+    - ripgrep
+    - libxkbfile  # [linux]
+    - xorg-libx11  # [linux]
+    - krb5 ==1.21.2  # [linux]
```

Comment:
Same comment about the [global pinnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/4cea2512b45b87086db6251e12c7bc233c53468c/recipe/conda_build_config.yaml#L430-L431). `krb` is covered there so it would be better not to override that. If you must, an explanation would help us make a decision.

Reply:
similar to other nodejs pinning comment, removing krb pinning since it's not needed now